src/ServiceClient/ISocketManager.cs

```diff
@@ -0,0 +1,9 @@
+using System.Net.WebSockets;
+
+namespace DeribitServiceClient.ServiceClient;
+
+public interface ISocketManager
+{
```

Comment:
I think that needs to encapsulate in Socket Manager the logic to manage client connection, and expose only and event with capture book an ticker data.